presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/MergeIdenticalWindows.java

```diff
@@ -97,7 +97,7 @@ public PlanNode visitWindow(
         {
             checkState(!node.getHashSymbol().isPresent(), "MergeIdenticalWindows should be run before HashGenerationOptimizer");
             checkState(node.getPrePartitionedInputs().isEmpty() && node.getPreSortedOrderPrefix() == 0, "MergeIdenticalWindows should be run before AddExchanges");
-            checkState(node.getFrames().values().size() == 1, "More than one frame per WindowNode is not yet supported");
+            checkState(node.getWindowFunctions().values().size() == 1, "More than one frame per WindowNode is not yet supported");
```

Comment:
I'll bring this over from the branch that dispenses with SpecificationAndFrame. It should be on this commit anyway.